src/components/project-pages/ConsciousShoppingPreview.tsx

```diff
@@ -1,69 +1,79 @@
-import React from "react";
-import styled from "styled-components";
 import {
   FlexboxProps,
-  flexbox,
-  layout,
+  GridProps,
   LayoutProps,
   PositionProps,
-  GridProps,
+  flexbox,
   grid,
+  layout,
+  position,
 } from "styled-system";
+
+import { Link } from "react-router-dom";
+import { PROJECTS_URL } from "../../constants/router-urls";
+import React from "react";
+import Timer from "../Timer";
 import bag from "../assets/conscious-shopping/bag.jpg";
-import projectIcon from "../assets/project-page/shell.png";
-import trex from "../assets/conscious-shopping/t-rex.jpg";
 import boots from "../assets/conscious-shopping/boots.jpg";
-import sunglasses from "../assets/conscious-shopping/sunglasses.jpg";
+import projectIcon from "../assets/project-page/shell.png";
 import shell from "../assets/conscious-shopping/concha.jpg";
-import Timer from "../Timer";
+import styled from "styled-components";
+import sunglasses from "../assets/conscious-shopping/sunglasses.jpg";
+import trex from "../assets/conscious-shopping/t-rex.jpg";
 import { zIndexes } from "../theme";
-import { Link } from "react-router-dom";
-import { PROJECTS_URL } from "../../constants/router-urls";
-
-type GridLayoutProps = FlexboxProps & LayoutProps & PositionProps & GridProps;
 
-const Grid = styled.div<GridLayoutProps>`
+const Div = styled.div<GridProps & LayoutProps & FlexboxProps>`
```

Comment:
could this extend `Flex` component?

Reply:
Not this time.. If you look on line 61, the div takes `flex` **and** `grid` as props which means that of we extended using `Flex` it would override the `grid` in the props array